src/Lykke.Service.B2c2Adapter/Modules/ServiceModule.cs

```diff
@@ -49,7 +51,9 @@ protected override void Load(ContainerBuilder builder)
                 .As<IStopable>()
                 .SingleInstance()
                 .WithParameter(TypedParameter.From(_settings.InstrumentLevels))
-                .WithParameter(TypedParameter.From(_settings.PublishFromCacheInterval));
+                .WithParameter("reconnectIfNeededInterval", _settings.ReconnectIfNeededInterval)
```

Comment:
(minor) seems like you can unite these parameters into a separate section/class.